src/timeline/media/css/main.css

```diff
@@ -1,174 +1,542 @@
-.transition_menu { cursor: pointer; float:left;margin-top:4px;margin-left:4px;margin-right:4px;width:12px;height:12px;background-image: url(../images/menu.png);}
-.transition_menu:hover { background-image: url(../images/menu_hover.png); }
-.transition_label { float:left; padding-top: 2px; width:50px; margin-left: 2px; color:#fff;font-weight:bold;}
-.highlight_transition { border-color: #32b7ea; }
+.transition_top {
+  background: -webkit-gradient(
+    linear, left top, left bottom,
+    color-stop(0%, rgba(255, 255, 255, 0.75)),
+    color-stop(100%, rgba(255, 255, 255, 0)));
+  border-radius: 8px;
+  overflow: hidden;
+}
+
+.transition_border {
+  display: block;
+  position: absolute;
+  /* Subtract combined L+R border widths */
+  width: calc(100% - 4px);
+  /* Subtract combined T+B border heights */
+  height: calc(100% - 2px);
+  border-radius: 8px;
+  border: 1px solid #0192c1;
+  border-left-width: 2px;
+  border-right-width: 2px;
+}
+
+.transition {
+  cursor: move;
+  z-index: 5000;
+  color: #fff;
+  min-width: 0;
+  height: 64px;
+  position: absolute;
+  margin-top: 0;
+  left: 100px;
+  font-size: 9pt;
+  background-color: rgba(#0192c1, 0.5);
+  background: -webkit-linear-gradient(top, #0192c1 0%, #3fa1bf 100%);
+  border-radius: 8px;
+  background-image: url(../images/transition.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 66px;
+  opacity: 0.75;
+  box-shadow: 0 0 10px #000;
+}
+
+.transition_label {
+  float: left;
+  padding-top: 2px;
+  width: 50px;
+  margin-left: 2px;
+  color: #fff;
+  font-weight: bold;
+}
+
+/*
+ * Timeline object Animations
+ */
+.timeline_obj.ng-enter,
+.timeline_obj.ng-leave {
+  -webkit-transition: 0.1s linear all;
+  transition: 0.1s linear all;
+}
 
-/* Transitions Animations */
-.transition.ng-enter,
-.transition.ng-leave {
-  -webkit-transition:0.1s linear all;
-  transition:0.1s linear all;
+.timeline_obj.ng-enter,
+.timeline_obj.ng-leave.ng-leave-active {
+  opacity: 0;
 }
 
-.transition.ng-enter,
-.transition.ng-leave.ng-leave-active {
-  opacity:0;
+.timeline_obj.ng-leave,
+.timeline_obj.ng-enter.ng-enter-active {
+  opacity: 1;
 }
-.transition.ng-leave,
-.transition.ng-enter.ng-enter-active {
-  opacity:1;
+
+/*
+ * Timeline object outlines (borders) change color to indicate state
+ */
+/* Regular unselected border, semi-transparent until hover */
+.timeline_obj>.tl-border {
+  border-color: #4b92ada8;
 }
 
+.timeline_obj:hover>.tl-border {
+  border-color: #4b92adff;
+}
+
+/* Selected item border, semi-transparent unless hovered */
+.ui-selected>.tl-border {
+  border-color: #ff0000a8;
+}
+
+.ui-selected:hover>.tl-border {
+  border-color: #ff0000ff;
+}
```

Comment:
Because the `<div tl-track>` is a container for `<div class="tl-border">`, which is in turn a container for all of the rest of its contents, my initial attempts at dicking around with setting `opacity` to create hover effects all failed. I couldn't change the opacity of the border `<div>` without everything inside it also becoming semi-transparent. So, I finally hit on the idea of just assigning semi-transparent _colors_ to the borders, in the non-hovered state.